src/App.tsx

```diff
@@ -13,17 +13,21 @@ import SelectToken from './components/SelectToken/SelectToken';
 import Stake from './components/Stake/Stake';
 import { Swap } from './pages/Swap/Swap';
 import { useChecks } from './utils';
+
 function App() {
   const [isInstalled, isConnected] = useChecks();
   const [selectTokenDrop, setSelectTokenDrop] = useState(false);
   const [stakeModal, setStakeModal] = useState(false);
+  const [isSelectError, setIsSelectError] = useState<string>('');
   const [checkedCoinTop, setCheckedCoinTop] = useState({
-    title: 'L-BTC',
+    title: 'L_BTC',
     image: 'liquid-btc.svg',
+    hash: '5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225',
   });
   const [checkedCoinBottom, setCheckedCoinBottom] = useState({
-    title: 'L-BTC',
-    image: 'liquid-btc.svg',
+    title: 'L_USDT',
+    image: 'liquid-tether.svg',
+    hash: '9b3d7db7f93cae59e9407232b9864b54c21cba20b992d707fa45136fbfa9b8fa',
```

Comment:
everything should be in a single place, ie. data.json. therwise we have two places to depends on asset hash

Reply:
fixed this